### PR TITLE
Improve docs of static routes

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1796

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/junos_static_routes.py [hunk 1]
```diff
@@ -136,18 +135,42 @@
...
       - afi: ipv4
         routes:
-        - dest: 10.200.16.75/24
+        - dest: 192.168.16.0/24
           next_hop:
-          - forward_router_address: 10.200.16.2
+          - forward_router_address: 172.16.1.2
     state: deleted
 
+# Task Output
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:143]:
```suggestion
# Task Output:
--------------
```

File: plugins/modules/junos_static_routes.py [hunk 2]
```diff
@@ -168,19 +190,42 @@
...
       - afi: ipv4
         routes:
-        - dest: 10.200.16.75/24
+        - dest: 192.168.16.0/24
           next_hop:
-          - forward_router_address: 10.200.16.2
+          - forward_router_address: 172.16.1.2
     state: merged
 
+# Task Output
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:198]:
```suggestion
# Task Output:
--------------
```

File: plugins/modules/junos_static_routes.py [hunk 3]
```diff
@@ -237,6 +308,35 @@
           - forward_router_address: 10.200.16.2
     state: replaced
 
+# Task Output
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:311]:
```suggestion
# Task Output:
--------------
```

File: plugins/modules/junos_static_routes.py [hunk 4]
```diff
@@ -201,17 +246,43 @@
     - address_families:
       - afi: ipv4
         routes:
-        - dest: 10.200.16.75/24
+        - dest: 192.168.16.0/24
           next_hop:
-          - forward_router_address: 10.200.16.2
+          - forward_router_address: 172.16.0.1
     state: overridden
+# Task Output
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:253]:
```suggestion
# Task Output:
--------------
```

**Review by rohitthakur2590 — CHANGES_REQUESTED** [review]

File: plugins/modules/junos_static_routes.py (hunk 1 above)
Comment (rohitthakur2590) [reply plugins/modules/junos_static_routes.py:143]: Lets follow the same format for before state and After state as we are doing for all the documentations now:
# Before state:
# -----------------

File: plugins/modules/junos_static_routes.py [hunk 5]
```diff
@@ -191,7 +236,7 @@
 # admin# show routing-options
 # static {
 #     route 192.168.47.0/24 next-hop 172.16.1.2;
-#     route 192.168.16.0/24 next-hop 172.16.0.1;
+#     route 192.168.16.0/24 next-hop 172.16.1.2;
 # }
 
 - name: Override provided configuration with device configuration (default operation
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:242]: Override running configuration with provided configuration

File: plugins/modules/junos_static_routes.py [hunk 6]
```diff
@@ -237,6 +308,35 @@
...
+#   - address_families:
+#       - afi: ipv4
+#         routes:
+#           - dest: 192.168.16.0/24
+#             next_hop:
+#               - forward_router_address: 172.16.1.2
+#           - dest: 192.168.47.0/24
+#             next_hop:
+#               - forward_router_address: 10.200.16.2
+
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:339]: Need to have examples for gathered, parsed and rendered .

**Review by rohitthakur2590 — CHANGES_REQUESTED** [review]

File: changelogs/fragments/fix_docs_static_route.yaml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+---
+doc_changes:
+  - Fix docs of static routes.
```
Comment (Qalthos) [inline changelogs/fragments/fix_docs_static_route.yaml:3]:
```suggestion
  - junos_static_routes - add task output to module documentation examples. (https://github.com/ansible-collections/junipernetworks.junos/pull/402)
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by rohitthakur2590** [comment]
recheck

File: plugins/modules/junos_static_routes.py [hunk 8]
```diff
@@ -246,7 +346,92 @@
...
+#       - afi: ipv4
+#         routes:
+#           - dest: 192.168.16.0/24
+#             next_hop:
+#               - forward_router_address: '[''172.16.1.2'', ''172.16.1.3'']'
+#           - dest: 192.168.47.0/24
+#             next_hop:
+#               - forward_router_address: 10.200.16.2
 """
 RETURN = """
```
Comment (rohitthakur2590) [inline plugins/modules/junos_static_routes.py:436]: could you also include documentation as below:  
rendered:
  description: The provided configuration in the task rendered in device-native format (offline).
  returned: when I(state) is C(rendered)
  type: list
  sample:
    - <nc:protocols xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0">
gathered:
  description: Facts about the network resource gathered from the remote device as structured data.
  returned: when I(state) is C(gathered)
  type: list
  sample: >
    This output will always be in the same format as the
    module argspec.
parsed:
  description: The device native config provided in I(running_config) option parsed into structured data as per module argspec.
  returned: when I(state) is C(parsed)
  type: list
  sample: >
    This output will always be in the same format as the
    module argspec.

**Pull request merged**